rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/180_update_dense_vector_type.yml

```
@@ -0,0 +1,1401 @@
+---
```

It seems like all these tests have the same `skip`. You can make this simpler with a skip in setup. 
```suggestion
setup:
  - skip:
      version: " - 8.14.99"
      reason: "vector index types became updatable in 8.15.0"
---
```